src/Components/Messages/Messages.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import MessageChatCard from "./MessageChatCard/MessageChatCard";
+import "./message.css";
+
+const Messages = () => {
+  const tempMessagesRoom = [
+    {
+      id: Math.round(Math.random() * 10000000),
+      title: "Pop",
+    },
+    {
+      id: Math.round(Math.random() * 10000000),
+      title: "EDM",
+    },
+    {
+      id: Math.round(Math.random() * 10000000),
+      title: "Electro",
+    },
+    {
+      id: Math.round(Math.random() * 10000000),
+      title: "Chill",
+    },
+  ];
```

Comment:
Will this be populated with the message rooms from the database?

Reply:
Yes, these are only created as placeholders.